Magento_Catalog/templates/product/list/addto/compare.phtml

```diff
@@ -2,13 +2,14 @@
 // @codingStandardsIgnoreFile
 /** @var $block Magento\Catalog\Block\Product\ProductList\Item\AddTo\Compare */
 ?>
-<button class="action tocompare button button--icon"
-        title="<?php echo $block->escapeHtml(__('Add to Compare')); ?>"
-        aria-label="<?php echo $block->escapeHtml(__('Add to Compare')); ?>"
-        data-post='<?php /* @escapeNotVerified */ echo $block->getCompareHelper()->getPostDataParams($block->getProduct()); ?>'
-        role="button"
+<button
+    class="catalog-grid-item__secondary-action"
+    title="<?php echo $block->escapeHtml(__('Add to Compare')); ?>"
+    aria-label="<?php echo $block->escapeHtml(__('Add to Compare')); ?>"
+    data-post='<?php /* @escapeNotVerified */ echo $block->getCompareHelper()->getPostDataParams($block->getProduct()); ?>'
+    role="button"
```

Comment:
Please change echo to shorthand to keep the same rules

Reply:
changed